sphinx_wagtail_theme/layout.html

```diff
@@ -24,7 +24,7 @@
         <title>{{ title|striptags|e }}{{ titlesuffix }}</title>
     {% endblock %}
     {%- if favicon_url %}
-        <link rel="shortcut icon" href="{{ pathto('_static/' + favicon_url, 1) }}" />
+        <link rel="shortcut icon" href="{{ pathto(favicon_url, 1) }}" />
```

Comment:
Changed this here to prevent the `_/static/_static/favicon.ico` url from being built

![Screenshot 2024-01-25 181035](https://github.com/wagtail/sphinx-wagtail-theme/assets/74812563/3f6a13d8-1732-49e3-8435-416ca1f5e818)